codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionDecoder.java

```diff
@@ -57,6 +57,8 @@
      */
     Http2FrameListener frameListener();
 
+    int requiredBytes();
```

Comment:
Thanks @Scottmitch. I wasn't sure whether we would ever expect users to implement these particular interfaces themselves and it doesn't break any _consumers_ of the API. But it can be worked around of course... just need some way to access through the layers of abstraction from `Http2ConnectionHandler`. At this point was mainly to demonstrate how http2 could benefit from this with minimal effort.

Reply:
@njhill yes please don't do this. We don't want to break the Api here.